### ISSUE TITLE
ASP.NET Core RC2 Web Application with no Authentication

### ISSUE THREAD

**Issue by rustd** [OmniSharp/generator-aspnet#693]
This is a close to final sample for RC2. Attaching the sample below
[WebApp_NoAuth.zip](https://github.com/OmniSharp/generator-aspnet/files/260049/WebApp_NoAuth.zip)

.NET CLI
 .NET Command Line Tools (1.0.0-rc2-002673) or higher
 Install Location: https://github.com/dotnet/cli#installers-and-binaries

NuGet Feed
https://www.myget.org/F/aspnetrelease/api/v3/index.json
https://api.nuget.org/v3/index.json

//cc @sayedihashimi

**Comment by peterblazejewicz** [comment]
this one fails during build:
```
 dotnet build
Project WebApplicationBasic (.NETCoreApp,Version=v1.0) will be compiled because expected outputs are missing
Compiling WebApplicationBasic for .NETCoreApp,Version=v1.0
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): warning NU1007: Dependency specified was Microsoft.AspNetCore.Mvc >= 1.0.0-rc2-20801 but ended up with Microsoft.AspNetCore.Mvc 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(16,81): warning NU1007: Dependency specified was Microsoft.Extensions.Configuration.EnvironmentVariables >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Configuration.EnvironmentVariables 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(17,65): warning NU1007: Dependency specified was Microsoft.Extensions.Configuration.Json >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Configuration.Json 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): warning NU1007: Dependency specified was Microsoft.Extensions.Logging.Console >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Logging.Console 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(20,60): warning NU1007: Dependency specified was Microsoft.Extensions.Logging.Debug >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Logging.Debug 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(7,58): error NU1001: The dependency Microsoft.AspNetCore.Diagnostics >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(9,42): error NU1001: The dependency Microsoft.AspNetCore.Razor.Tools >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(15,58): error NU1001: The dependency Microsoft.AspNetCore.StaticFiles >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(21,72): error NU1001: The dependency Microsoft.VisualStudio.Web.BrowserLink.Loader >= 14.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Http.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Http.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Http.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.WebUtilities >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.WebUtilities >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.WebUtilities >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(18,54): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(20,60): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(18,54): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(20,60): error NU1001: The dependency Microsoft.Extensions.Logging.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Http.Extensions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Server.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Hosting.Server.Abstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(17,65): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(17,65): error NU1001: The dependency Microsoft.Extensions.FileProviders.Physical >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20801 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): error NU1001: The dependency Microsoft.Extensions.PlatformAbstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.Net.Http.Headers >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.Net.Http.Headers >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Net.Http.Headers >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.FileProviders.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.FileSystemGlobbing >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Diagnostics.Abstractions >= 1.0.0-rc2-20825 could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.Extensions.WebEncoders >= 1.0.0-rc2-20825 could not be resolved.

Compilation failed.
    5 Warning(s)
    59 Error(s)
```
```
dotnet --info
.NET Command Line Tools (1.0.0-preview1-002700)

Product Information:
 Version:     1.0.0-preview1-002700
 Commit Sha:  0336f6bb34

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
```

**Comment by rustd** [comment]
Can you please try with CLI version 1.0.0-rc2-002673?

**Comment by rustd** [comment]
Also you can replace the build number in the packages with *

**Comment by rustd** [comment]
I am uploaded the final RC2 template here.
[WebApp_NoAuth.zip](https://github.com/OmniSharp/generator-aspnet/files/263600/WebApp_NoAuth.zip)

Please use only the following feed and CLI version

**.NET CLI**
.NET Command Line Tools 1.0.0-preview1-002702
Install Location: https://github.com/dotnet/cli#installers-and-binaries

**NuGet Feed**
https://www.myget.org/F/aspnetrc2/api/v3/index.json
https://api.nuget.org/v3/index.json

**Comment by peterblazejewicz** [comment]
The PR bits are updated, but event with 1.0.0-preview1-002702 and updated feeds the restore and build fails on my machine:
```
log  : Restore failed in 10236ms.

Errors in /Users/piotrblazejewicz/development/WebApplicationBasic/project.json
    Unable to resolve 'NuGet.Versioning (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Packaging (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Frameworks (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Versioning (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Packaging (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Frameworks (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Packaging (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Versioning (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.
    Unable to resolve 'NuGet.Frameworks (>= 3.5.0-beta-final)' for '.NETCoreApp,Version=v1.0'.

NuGet Config files used:
    /Users/piotrblazejewicz/development/WebApplicationBasic/NuGet.Config
    /Users/piotrblazejewicz/.nuget/NuGet/NuGet.Config

Feeds used:
    https://api.nuget.org/v3/index.json
    https://www.myget.org/F/aspnetrc2/api/v3/index.json

Installed:
    61 package(s) to /Users/piotrblazejewicz/development/WebApplicationBasic/project.json
➜  WebApplicationBasic dotnet build
Project WebApplicationBasic (.NETCoreApp,Version=v1.0) will be compiled because expected outputs are missing
Compiling WebApplicationBasic for .NETCoreApp,Version=v1.0
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(7,58): error NU1001: The dependency Microsoft.AspNetCore.Diagnostics >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(8,50): error NU1001: The dependency Microsoft.AspNetCore.Mvc >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(9,42): error NU1001: The dependency Microsoft.AspNetCore.Razor.Tools >= 1.0.0-preview1-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(13,68): error NU1001: The dependency Microsoft.AspNetCore.Server.IISIntegration >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(14,61): error NU1001: The dependency Microsoft.AspNetCore.Server.Kestrel >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(15,58): error NU1001: The dependency Microsoft.AspNetCore.StaticFiles >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(16,81): error NU1001: The dependency Microsoft.Extensions.Configuration.EnvironmentVariables >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(19,62): error NU1001: The dependency Microsoft.Extensions.Logging.Console >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(20,60): error NU1001: The dependency Microsoft.Extensions.Logging.Debug >= 1.0.0-rc2-final could not be resolved.
/Users/piotrblazejewicz/development/WebApplicationBasic/project.json(21,72): error NU1001: The dependency Microsoft.VisualStudio.Web.BrowserLink.Loader >= 14.0.0-rc2-final could not be resolved.

Compilation failed.
    0 Warning(s)
    10 Error(s)

Time elapsed 00:00:00.0484629
```

**Comment by sayedihashimi** [comment]
I think the feed us missing some packages. I think it will be fixed soon.

**Comment by peterblazejewicz** [comment]
OK. Just ping me. The bits are updated in PRs

**Comment by rustd** [comment]
The packages are uploaded to this feed https://www.myget.org/F/aspnetrc2/api/v3/index.json

**Comment by oldnerd** [comment]
Built fine for me on a fresh dotnet install,  Ran properly.

**Comment by peterblazejewicz** [comment]
@oldnerd Thanks!

**Comment by peterblazejewicz** [comment]
Updated/merged

Closed via: ae4bc68
